source/documentation/continuous-integration-and-delivery/index.md

```diff
@@ -0,0 +1,9 @@
+# Continuous Integration and Delivery
+
+The Reliability Engineering team uses [Concourse](https://concourse-ci.org) for testing and deploying code to production.
```

Comment:
I'd like to see a link to the actual concourse in the top section of this page please

Reply:
Done in f574de1